project/settings_hls.tcl

```diff
@@ -19,8 +19,9 @@ switch -glob -- $exe {
     }
     default {
         # Set the FPGA part number
-        set_part {xcvu7p-flvb2104-1-e} -tool vivado
+        #set_part {xcvu7p-flvb2104-1-e} -tool vivado
         #set_part {xcku115-flvb2104-1-e} -tool vivado
+        set_part {xcvu9p-flgc2104-1-e} -tool vivado
```

Comment:
Is there a reason for switching to a VU9P?

Reply:
I just wanted to try with a different FPGA to see if the resource usage was a bit better, but it was about the same as a VU7P.